nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -38,6 +38,17 @@ In addition to numerous new and upgraded packages, this release has the followin
   for a transition period so that in time the ecosystem can switch without
   breaking compatibility with any supported NixOS release.
 
+- `nscd` functionality, necessary to provide non-glibc-builtin NSS
+  modules (such as `nss_systemd` or `nss_ldap`) can now be provided by
+  `nsncd`, by setting `services.nscd.enableNsncd` to `true`.
+
+  The glibc-provided `nscd` daemon wasn't very reliable, and it was not fully
+  possible to disable caching functionality, causing various issues and stuck
+  lookups.
+
+  The plan is to switch from `glibc-nscd` to `nsncd` by default for the release
+  after this.
```

Comment:
These subpoints should be prefixed with `-` (I think).

Reply:
Makes sense. Updated.